make/program

```diff
@@ -2,6 +2,7 @@
 # Models (to be passed through stanc)
 ##
 CMDSTAN_MAIN ?= src/cmdstan/main.cpp
+CMDSTAN_MAIN_O ?= src/cmdstan/main.o
```

Comment:
Please use the patsubst command to derive from CMDSTAN_MAIN the value for CMDSTAN_MAIN_O.